src/Greenshot.Editor/Forms/ImageEditorForm.cs

```diff
@@ -280,11 +284,6 @@ private void UpdateUi()
             // a smaller size than the initial panel size (as set by the forms designer)
             panel1.Height = 10;
 
-            fontFamilyComboBox.PropertyChanged += FontPropertyChanged;
```

Comment:
I must be blind, but where did these lines go?? Did I remove them, don't recall? I do understand that this is bad, register events in an update ui doesn't sound good, but where did they go?

Reply:
I found that you moved these to the Designer file, that makes sense.